server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -587,13 +598,14 @@ public InitialScan startScan(TInfo tinfo, TCredentials credentials, TKeyExtent t
       if (tablet == null)
         throw new NotServingTabletException(textent);
 
-      Set<Column> columnSet = new HashSet<>();
+      HashSet<Column> columnSet = new HashSet<>();
```

Comment:
Could be: `columns.stream().map(tcol -> new Column(tcol)).collect(Collectors.toSet());`